tests/dailytest/case.go

```diff
@@ -163,6 +163,9 @@ func RunCase(src *sql.DB, dst *sql.DB, schema string) {
 	tr.run(caseTblWithGeneratedCol)
 	tr.execSQLs([]string{"DROP TABLE gen_contacts;"})
 
+	tr.run(caseCreateView)
+	tr.execSQLs([]string{"DROP TABLE base_for_view;"})
```

Comment:
```
282 [2019/05/23 11:23:06.150 +08:00] [DEBUG] [schema.go:170] ["drop table success"] [name=v] [id=41]
283 [2019/05/23 11:23:06.150 +08:00] [DEBUG] [load.go:244] ["exec ddl"] [ddl="{\"Database\":\"test\",\"Table\":\"v\",\"SQL\":\"drop view v\"}"]
284 [2019/05/23 11:23:06.155 +08:00] [ERROR] [load.go:235] ["parse sql failed"] [sql="drop view v"] [error="line 1 column 11 near \"\" (total length 11)"] [errorVerbose="line 1 column 11 near \"\" (total length 11)\ngithub.com/pingcap/errors.AddStack\n\t/Users/huang    jiahao/go/pkg/mod/github.com/pingcap/errors@v0.11.1/errors.go:174\ngithub.com/pingcap/errors.Trace\n\t/Users/huangjiahao/go/pkg/mod/github.com/pingcap/errors@v0.11.1/juju_adaptor.go:15\ngithub.com/pingcap/parser.(*Parser).Parse\n\t/Users/huangjiahao/go/pkg/mod/g    ithub.com/pingcap/parser@v0.0.0-20190411060258-7e41749fa69c/yy_parser.go:117\ngithub.com/pingcap/parser.(*Parser).ParseOneStmt\n\t/Users/huangjiahao/go/pkg/mod/github.com/pingcap/parser@v0.0.0-20190411060258-7e41749fa69c/yy_parser.go:136\ngithub.com/pingcap/tidb    -binlog/pkg/loader.isCreateDatabaseDDL\n\t/Users/huangjiahao/go/src/github.com/pingcap/tidb-binlog/pkg/loader/load.go:233\ngithub.com/pingcap/tidb-binlog/pkg/loader.(*loaderImpl).execDDL.func1\n\t/Users/huangjiahao/go/src/github.com/pingcap/tidb-binlog/pkg/loade    r/load.go:252\ngithub.com/pingcap/tidb-binlog/pkg/util.RetryOnError\n\t/Users/huangjiahao/go/src/github.com/pingcap/tidb-binlog/pkg/util/util.go:143\ngithub.com/pingcap/tidb-binlog/pkg/loader.(*loaderImpl).execDDL\n\t/Users/huangjiahao/go/src/github.com/pingcap/    tidb-binlog/pkg/loader/load.go:246\ngithub.com/pingcap/tidb-binlog/pkg/loader.(*batchManager).execDDL\n\t/Users/huangjiahao/go/src/github.com/pingcap/tidb-binlog/pkg/loader/load.go:519\ngithub.com/pingcap/tidb-binlog/pkg/loader.(*batchManager).put\n\t/Users/huan    gjiahao/go/src/github.com/pingcap/tidb-binlog/pkg/loader/load.go:538\ngithub.com/pingcap/tidb-binlog/pkg/loader.(*loaderImpl).Run\n\t/Users/huangjiahao/go/src/github.com/pingcap/tidb-binlog/pkg/loader/load.go:409\ngithub.com/pingcap/tidb-binlog/drainer/sync.(*My    sqlSyncer).run\n\t/Users/huangjiahao/go/src/github.com/pingcap/tidb-binlog/drainer/sync/mysql.go:117\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1337"]
285 [2019/05/23 11:23:06.168 +08:00] [INFO] [load.go:269] ["exec ddl success"] [sql="drop view v"]
286 [2019/05/23 11:23:06.168 +08:00] [DEBUG] [load.go:186] ["markSuccess txns"] ["txns len"=1]
287 [2019/05/23 11:23:06.168 +08:00] [INFO] [syncer.go:241] ["write save point"] [ts=408572863715999745]
```
please update parser dep too, the current version can't support drop view although it can ignore this  error now.

Reply:
Which version of parser to use?

Reply:
check master branch of TiDB (go.mod)

Reply:
@july2993 Build failed after upgrading to parser@v0.0.0-20190521133604-4afa585be9a4